packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SingleImageViewer/components/ZoomHelperOverlay.js

```diff
@@ -0,0 +1,78 @@
+import { Box, Text } from 'grommet'
+import { bool, string } from 'prop-types'
+import styled, { keyframes } from 'styled-components'
+
+import { useTranslation } from '@translations/i18n'
+
+const fadeIn = keyframes`
+  from { opacity: 0; }
+  to { opacity: 1; }
+`
+
+const fadeOut = keyframes`
+  from { opacity: 1; }
+  to { opacity: 0; }
+`
+
+const StyledOverlay = styled(Box)`
+  position: absolute;
+  top: 0;
+  background: ${props => props.theme.dark ? 'rgba(45, 45, 45, 0.40)' : 'rgba(255, 255, 255, 0.40)'};
+  backdrop-filter: blur(1px);
+  pointer-events: none;
+  animation: ${props => props.$fadeOut ? fadeOut : fadeIn} 0.3s forwards;
+`
+
+const StyledBox = styled(Box)`
+  background: ${props => props.theme.dark ? 'rgba(45, 45, 45, 0.60)' : 'rgba(255, 255, 255, 0.60)'};
+  backdrop-filter: blur(4px);
+  position: sticky;
+  top: ${props => props.$top};
```

Comment:
This Box is not sticking, likely because the classifier layout already includes sticky elements. For instance, when I visit i-fancy-cats, a portrait-oriented subject loads and the StyledBox is at 240px from the top of the subject viewer, and does not move with scroll. I think (just my opinion) that a sticky message would be an unpleasant UX.

Instead of adding `position: sticky` complexity to this feature, I recommend simply center-aligning StyledBox both horizontally and vertically within StyledOverlay. It will increase the chance of a volunteer seeing the message regardless of subject size and aspect ratio.

Reply:
Good suggestion. Center aligning the message also simplifies the code quite a bit as detailed in [d6e171d](https://github.com/zooniverse/front-end-monorepo/pull/6805/commits/d6e171db5618f2f87d72142f3114e01c23790308). I've changed the alignment accordingly and we can revisit if necessary.